radicle-cli/src/commands/remote/add.rs

```diff
@@ -0,0 +1,38 @@
+use radicle::node::tracking::store::Config;
+use radicle::{git::Url, node::TRACKING_DB_FILE, prelude::Id, Profile};
+use radicle_crypto::PublicKey;
+
+use crate::git::rad_add_remote;
+use crate::{git, terminal as term};
+
+pub fn run(
+    repository: &git::Repository,
+    profile: &Profile,
+    pubkey: &PublicKey,
+    name: Option<String>,
+    id: Id,
+) -> anyhow::Result<()> {
+    let name = name
+        .map_or(get_remote_name(profile, pubkey)?, Some)
+        .ok_or(anyhow::anyhow!("an alias need to be specified"))?;
+
+    if git::is_remote(repository, &name)? {
+        anyhow::bail!("remote with alias `{name}` already exists");
+    }
+    let url = Url::from(id).with_namespace(*pubkey);
+    let url = url.to_string();
+    let remote = rad_add_remote(repository, &name, &url)?;
+    term::success!(
+        "Remote {} added with {}",
+        remote.name,
+        remote.url.to_string()
```

Comment:
We already have the string version of the `Url`
```suggestion
        url
```